packages/components/src/stories/PenrosePrograms.tsx

```diff
@@ -1,12 +1,12 @@
 export const oneSet = {
```

Comment:
Seems like there's a way to reference to static files in storybook. Perhaps we can just load files from `examples` directly: https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook-configuration.